### DISCUSSION TITLE
AERODYN v15.00.* BLADE DEFINITION INPUT FILE

### DISCUSSION THREAD

**Discussion by sjihf12** [OpenFAST/openfast#1763] [Q&A]
Hello, I would like to ask if I change the 'BITwist' and 'BIChord' in the 'AERODYN v15.00.* BLADE DEFINITION INPUT FILE', other parameters such as 'BICrvAC' and 'BISwpAC' need to be modified? If other parameters need to be modified, how to calculate them by OPENFAST?

**Comment by jjonkman** [comment]
Dear @sjihf12,

I would generally expect that `BlTwist` and `BlChord` to be specified independently from other parameters in the AeroDyn blade input file.

Best regards,

**Comment by sjihf12** [comment]
Thank you!
